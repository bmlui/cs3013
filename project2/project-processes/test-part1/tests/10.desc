evaluate behavior with unknown command 
