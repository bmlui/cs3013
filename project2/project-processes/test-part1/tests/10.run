PATH=/bad ./wshell < tests/10.in
