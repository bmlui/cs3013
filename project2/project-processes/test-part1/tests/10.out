test-part1$ laksdjflkasjdflkjsd
wshell: could not execute command: laksdjflkasjdflkjsd
test-part1$ exit
